target/linux/ath79/nand/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -9,6 +9,10 @@ board=$(board_name)
 case "$FIRMWARE" in
 "ath10k/cal-pci-0000:00:00.0.bin")
 	case $board in
+	domywifi,dw33d)
+		caldata_extract "art" 0x5000 0x844
+		ath10k_patch_mac $(mtd_get_mac_binary art 0x12)
```

Comment:
With mtd_get_mac_binary being used here, can this be replaced with mtd_mac_address in DTS? Or is this not possible with ath10k?

Reply:
I tried mtd_mac_address but could not get the correct mac address.

Reply:
Note that is has to be a property of wifi@0,0:
```
&pcie0 {
   wifi@0,0 {
      ...
      mtd-mac-address = <&art 0x12>;
   };
};
```
(No offense if you already did it correctly...)

Reply:
I don't know how to use mtd_mac_address to get the address in dts, I will try again, thank you.

Reply:
Note that it might still not work, but I'm really interested to find out.

Reply:
`&pcie0 {
	status = "okay";

	wifi@0,0 {
		compatible = "qcom,ath10k";
		reg = <0 0 0 0 0>;
		mtd-mac-address = <&art 0x12>;
	};
};`

Still can't get the correct mac address

Reply:
Okay, thanks for testing!